src/Extension/ExtensionManager.php

```diff
@@ -138,10 +138,18 @@ public function disable($name)
         $enabled = $this->getEnabled();
 
         if (($k = array_search($name, $enabled)) !== false) {
-            unset($enabled[$k]);
-
             $extension = $this->getExtension($name);
 
+            if (in_array('locale', $extension->keywords)) {
+                $default_locale = $this->app->make('flarum.settings')->get('default_locale');
+                $locale_code = array_get($extension->extra, 'flarum-locale.code');
+                if (strcmp($default_locale, $locale_code) == 0) {
```

Comment:
Definitely!

I'm a newbie to php, and thought == would compare the pointer/reference so I looked for the strcmp() function.
Thank you for pointing out things like '==='! That's news to me